### ISSUE TITLE
Accessing the runner / may lead to exception

### ISSUE THREAD

**Issue by ArnaudBuchholz** [ArnaudBuchholz/ui5-test-runner#94]
Reproduced, this is because the following mapping matches `/` :
```javascript
{
  // Project mapping
  match: /^\/(.*)/,
  cwd: job.webapp,
  file: '$1',
  static: !job.watch && !job.debugDevMode
},
```
But after interpolation, `$1` is empty and it fails the file handler :
```
let filePath = /([^?#]+)/.exec(redirect)[1] // filter URL parameters & hash
```
because it does not match, it might be interesting to replace + with *

**Comment by ArnaudBuchholz** [comment]
https://github.com/ArnaudBuchholz/reserve/issues/99

**Comment by ArnaudBuchholz** [comment]
Fixed in `reserve@2.0.2`
